admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -390,7 +390,7 @@ var featuresCollection = (function() {
 
         modalConfirmation.create(translate_javascripts['Are you sure to delete this?'], null, {
           onContinue: function() {
-            _this.parent().parent().parent().remove();
+            _this.parent().parent().remove();
```

Comment:
`.parent().parent().parent().parent().parent().parent().parent().parent().parent()`.

Using a selector as a parameter of `parent()` could prevent another regression as soon as we update the DOM. Have you checked that is possible?

Reply:
You're right => I've changed to `_this.closest('.product-feature').remove();` but GitHub doesnt seems to publish my modifications...